odl/contrib/tomo/elekta.py

```diff
@@ -190,6 +193,158 @@ def elekta_icon_fbp(ray_transform,
     return fbp_op
 
 
+def elekta_xvi_geometry(sad=1000.0, sdd=1500.0,
+                        piercing_point=(512.0, 512.0),
+                        angles=np.linspace(0, 2 * np.pi, 650),
+                        detector_shape=(1024, 1024)):
+    """Tomographic geometry of the Elekta XVI system.
+
+    All measurments are given in millimeters unless otherwise stated.
```

Comment:
Perhaps there's a reference (like a link) to the system description that you can provide?

Reply:
It seems not, I'm trying to find one. This is only mentioned in derived articles (e.g. some medical physisist has measured it)

Reply:
Maybe this:
https://www.elekta.com/radiotherapy/treatment-solutions/imaging/xvi.html
??

Reply:
Says nothing about the geometry :/

Reply:
That's harder of course. Still nice to have something to click on, better than nothing.